Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.py

```diff
@@ -190,7 +223,8 @@ def fetch_incidents_high_risk_users(client: Client, max_results: int, last_run:
                 'occurred': timestamp_to_datestring(incident_created_time_ms),
                 'rawJSON': json.dumps(record1)
             }
-            incidents.append(inc1)
+            if record1.get('employeeId') is not None:
```

Comment:
can you explain this condition? will this cause for cases that were fetched before not to be fetched anymore? is this a bug fix?

Reply:
Previously blank records creating when API will not return any result,
to stop this we added extra condition so it will not create blank records